f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml

```diff
@@ -17,7 +17,6 @@
         <element label="Funding Plans" max_length="4000" name="funding_plans" reference="x_g_dis_atat_funding_plan" type="glide_list"/>
         <element label="Funding Request" max_length="32" name="funding_request" reference="x_g_dis_atat_funding_request" type="reference"/>
         <element label="Funding Requirement" max_length="32" name="funding_requirement" reference="x_g_dis_atat_funding_requirement" type="reference"/>
-        <element label="GFE Overview" max_length="32" name="gfe_overview" reference="x_g_dis_atat_gfe_overview" type="reference"/>
         <element label="IDIQ CLINs" max_length="4000" name="idiq_clins" reference="x_g_dis_atat_idiq_clin" type="glide_list"/>
```

Comment:
These changes appears to confirm the deletion of the `gfe_overview` column in the package. This was unexpected but appears to be a default behavior of SNOW. It was only noticed after looking for the `gfe_overview` column to delete in the `Acq. Package`.